packages/react-components/src/components/chart/hooks/useSync.ts

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from 'react';
+import { handleSyncChanges } from '../utils/handleSyncTrendCursors';
+import { getInstanceByDom } from 'echarts';
+import { getNewTrendCursor, ondragUpdateTrendCursor } from '../utils/getTrendCursor';
+import { calculateXFromTimestamp } from '../utils/getInfo';
+import useDataStore from '../../../store';
+import { UseSyncProps } from '../types';
+
+const useSync = ({
```

Comment:
What is the purpose of `useSync`?

Reply:
this only handles the changes received in the sync mode, let me add some comments